### ISSUE TITLE
Insert module needs an additional hint

### ISSUE THREAD

**Issue by dliberat** [evanlucas/learnyoumongo#59]
The directions for this lesson are quite vague. The insert code was easy enough to write, but it took well over an hour to figure out that the solution expects you to log the JSON data to the console AND NOTHING ELSE.

An additional hint, something along the lines of "Note: Do not log anything to the console except the JSON object or your solution will not pass the verification." would be very much appreciated.

**Comment by taylor-knapp** [comment]
I agree this module could use some clarification. I struggled with the following (much of which was  due to my own understanding):

1. The instructions lead me to believe that to insert into mongoDB I needed to use JSON.stringify(), which I've since read isn't true (I probably interpreted what was written incorrectly). Mongo receives objects themselves. The JSON.stringify() in this example is only to be passed to the console.log() for verification.

2. Unlike other lessons in learnyoumongo or learnyounode, when I attempted to run the program.js file and see how I was doing via print outs, it never returned anything (i.e. I had a few console.logs buried in the script to try and see where it was failing, and I wasn't able to see them using 'learnyoumongo run program.js'). This extends to actually trying to verify - instead of showing me what I was producing vs. what was expected, it just said failed. Those clues help.

Perhaps I'll work on my confidence and then take a stab at some of these updates.

**Comment by ValygarS** [comment]
Same, after reading the hints I was thinking, I have to use JSON.stringify() to insert into DB and not for the output to console. Thanks for help
